network/src/logging.rs

```diff
@@ -8,39 +8,69 @@
 //! use libra_config::network_id::NetworkContext;
 //! use libra_logger::prelude::*;
 //! use network::logging::*;
+//! use network::connectivity_manager::DiscoverySource;
+//! use futures::io::ErrorKind;
+//! use libra_network_address::NetworkAddress;
+//! use libra_types::PeerId;
 //!
 //! info!(
-//!   network_log(network_events::CONNECTIVITY_MANAGER_LOOP, &NetworkContext::mock())
-//!     .data(network_events::TYPE, network_events::START)
-//!     .field(network_events::EVENT_ID, &5)
+//!   NetworkSchema::new(&NetworkContext::mock())
+//!     .remote_peer(PeerId::random())
+//!     .network_address(&Some(NetworkAddress::mock())),
```

Comment:
```suggestion
//!     .remote_peer(&PeerId::random())
//!     .network_address(&NetworkAddress::mock()),
```

Although you might want to double check that you won't end up with those being dropped too early